### PR TITLE
Exclude js and github actions from changelog

### PR DESCRIPTION
**Pull request by marianeum**
JS & github actions dependencies generally don't affect anything for the consumers, so we shouldn't include them in the changelog. The only exception to this is backpack dependencies which include token changes. these should be part of the changelog, so creating a separate config for them.

+ [x] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-android/blob/main/CONTRIBUTING.md)

Remember to include the following changes:
+ [x] Component `README.md`
+ [x] Tests
+ [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/main/CODE_REVIEW_GUIDELINES.md)

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -21,6 +21,21 @@ updates:
...
+- package-ecosystem: npm
+  directory: "/"
+  schedule:
+    interval: daily
+    time: "10:00"
+  open-pull-requests-limit: 10
+  versioning-strategy: increase-if-necessary
+  labels:
+    - "bpk"
+    - "depencdencies"
```
Comment (olliecurtis) [inline .github/dependabot.yml:35]:
```suggestion
    - "dependencies"
```

Comment (marianeum) [reply .github/dependabot.yml:35]: whoops, thanks!

**Comment by marianeum** [comment]
argh looks like this is actually not working :(
> The property '#/updates/3' is a duplicate. Update configs must have a unique combination of 'package-ecosystem', 'directory', and 'target-branch'

File: .github/workflows/pr.yml [hunk 2]
```diff
@@ -17,3 +17,18 @@ jobs:
...
+    runs-on: ubuntu-latest
+    if: ${{ github.actor == 'dependabot[bot]' }}
+    steps:
+      - name: Fetch Dependabot metadata
+        id: dependabot-metadata
+        uses: dependabot/fetch-metadata@v1.3.1
+        with:
+          github-token: "${{ secrets.GITHUB_TOKEN }}"
+      - name: Add bpk label
+        if: contains('bpk-', steps.dependabot-metadata.outputs.dependency-names[0])
```
Comment (marianeum) [inline .github/workflows/pr.yml:31]: is this syntax correct? we will find out...

File: .github/workflows/pr.yml [hunk 3]
```diff
@@ -17,3 +17,18 @@ jobs:
...
+    runs-on: ubuntu-latest
+    if: ${{ github.actor == 'dependabot[bot]' }}
+    steps:
+      - name: Fetch Dependabot metadata
+        id: dependabot-metadata
+        uses: dependabot/fetch-metadata@v1.3.1
+        with:
+          github-token: "${{ secrets.GITHUB_TOKEN }}"
+      - name: Add bpk label
+        if: contains(steps.dependabot-metadata.outputs.dependency-names[0], 'bpk-')
```
Comment (marianeum) [inline .github/workflows/pr.yml:31]: is this syntax correct? we will find out...

**Review by frugoman — APPROVED** [review]

**Pull request merged**